src/p256.rs

```diff
@@ -126,10 +127,10 @@ fn montgomery_ladder(k: Scalar, init: Jacobian) -> Jacobian {
 }
 
 pub fn point_mul(k: Scalar) -> Affine {
-    let base_point = Jacobian(
-        FieldElement::from("6B17D1F2E12C4247F8BCE6E563A440F277037D812DEB33A0F4A13945D898C296"),
-        FieldElement::from("4FE342E2FE1A7F9B8EE7EB4A7C0F9E162BCE33576B315ECECBB6406837BF51F5"),
-        FieldElement::from(1),
+    let base_point = (
```

Comment:
If we want to use the unsigned_int create we should add these functions.

Reply:
Which functions ? `FieldElement::from_hex` and `FieldElement::from_literal` are functions defined in the `abstract_integers` crate.

Reply:
Right, but using `from` looks nicer  :slightly_smiling_face:. I think we should use `from` where possible to limit the number of functions we need. (Having explicit `from_XXX` is nice but also pollutes the API.)